JetUtilities/interface/ClosureMaker.hh

```diff
@@ -83,27 +83,28 @@ public:
 	void 				checkResponse();
 	pair<double,double> determineCanvasRange(double xmin, double xmax);
 	void 				makeCanvases();
-	void 				makeMergedCanvas();
+	void 				makeMergedCanvas(bool finemerge);
```

Comment:
Any way to fix the spacing here? I know it's not your fault, but this would be a good opportunity to standardize the use of spaces and tabs.

Reply:
I am converting all tabs to spaces when I modify a file now.